mkdocs.yml

```diff
@@ -0,0 +1,2 @@
+theme:
```

Comment:
This file is missing one required parameter and lots of optional ones, which you can see listed on [this](https://www.mkdocs.org/user-guide/configuration/) documentation page for MkDocs. I'd suggest going through what configuration you can define now, so that less work is needed in the future for integrating real documentation.